README.rst

```diff
@@ -189,11 +189,13 @@ You can also specify which framework bindings to build. The default is pytorch o
   # Build with all bindings (Pytorch, TF, Jax)
   NVTE_FRAMEWORK=all pip install .
 
-User Guide
-----------
+User Guide and Examples
+-----------------------
 
-For examples, tutorials and API reference please refer to the
-`User Guide <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/index.html>`_.
+For examples, tutorials and API reference please refer to:
+* `User Guide <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/index.html>`_ for the last release.
+* `Development User Guide <https://nvidia.github.io/TransformerEngine/>`_ for the development version.
+* `Examples <https://github.com/NVIDIA/TransformerEngine/tree/main/examples>`_.
```

Comment:
```suggestion
For examples, tutorials and API reference please refer to:

* `User Guide <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/index.html>`_ for the last release.
* `Development User Guide <https://nvidia.github.io/TransformerEngine/>`_ for the development version.
* `Examples <https://github.com/NVIDIA/TransformerEngine/tree/main/examples>`_.
```

Reply:
Maybe this should help. My comment was regarding the bullets not showing in the README, and it all just is in a single line.